packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -198,3 +199,88 @@ with 3 API mapping resources across different APIs and Stages.
 | api | $default  |   `https://${domainName}/foo`  |
 | api | beta  |   `https://${domainName}/bar`  |
 | apiDemo | $default  |   `https://${domainName}/demo`  |
+
+
+### Private Integration
+
+The `VpcLink` and private integration constructs can be used for integration of `HTTP` APIs with resource behind an Amazon VPC.
+
+#### VPC Link
+
+The following code uses `VpcLink` to connect an HTTP API endpoint to resources in a VPC - 
+
+```ts
+const vpc = new ec2.Vpc(stack, 'VPC');
+const vpcLink = new VpcLink(stack, 'VpcLink', { vpc });
+```
+
+Any existing `VpcLink` resource can be imported into the CDK app via the `VpcLink.fromVpcLinkId()`.
+
+```ts
+const stack = new Stack(app, 'my-stack');
+
+const awesomeLink = VpcLink.fromVpcLinkId(stack, 'awesome-vpc-link', 'us-east-1_oiuR12Abd');
+```
+
+#### ApplicationLoadBalancerPrivateIntegration
+
+The following code is a basic application load balancer private integration of HTTP API:
```

Comment:
This and the following sections should go into the readme for the new module `aws-apigatewayv2-integrations`.

Reply:
Done!